corral/bundle/bundle.pony

```diff
@@ -7,25 +7,24 @@ primitive BundleFile
   """
   Loader and creator of Bundle files.
   """
-  fun find_bundle_dir(env: Env, log: Log): (FilePath | None) =>
-    let cwd = Path.cwd()
-    var dir = cwd
-    while dir.size() > 0 do
-      log.fine("Looking for " + Files.bundle_filename() + " in: '" + dir + "'")
+  fun find_bundle_dir(env: Env, path: String, log: Log): (FilePath | None) =>
```

Comment:
If this parameter were `dir` that would be more consistent, and then most of the code below would be the same.